tests/cmd/stability/main.go

```diff
@@ -138,15 +151,15 @@ func main() {
 		oa.DumpAllLogs(operatorCfg, allClusters)
 	}()
 
+	// clean and deploy operator
+	oa.CleanOperatorOrDie(operatorCfg)
+	oa.DeployOperatorOrDie(operatorCfg)
+
```

Comment:
why move them here?

Reply:
the webhook delete is follow operator, if the test is terminate the webhook will not be clean. next run can't delete pod.

Reply:
Got it.